lib/models/user-whitelist.js

```diff
@@ -13,4 +14,28 @@ util.inherits(UserWhitelist, Base);
 
 require('../extend-with-factories')(UserWhitelist);
 
+UserWhitelist.prototype.isInTrial = function () {
+  return moment(this.attrs.trialEnd) > moment().utc();
+};
+
+UserWhitelist.prototype.isInGrace = function () {
+  return !this.isInTrial() && moment(this.attrs.gracePeriodEnd) > moment().utc();
+};
+
+UserWhitelist.prototype.isInActivePeriod = function () {
+  return moment(this.attrs.activePeriodEnd) > moment().utc();
+};
+
+UserWhitelist.prototype.isGraceExpired = function () {
+  return !this.isInTrial() && moment.utc(this.attrs.gracePeriodEnd) < moment().utc();
```

Comment:
You use moment.utc for gracePeriodEnd here but not above, I assume all the set periods are already in UTC.